coremltools/test/test_multiple_images_preprocessing.py

```diff
@@ -55,9 +55,9 @@ def compare_models(caffe_preds, coreml_preds):
         if relative_error > max_relative_error:
             max_relative_error = relative_error
 
-    print 'maximum relative error: ', max_relative_error
-    #print 'caffe preds : ', caffe_preds
-    #print 'coreml preds: ', coreml_preds      
+    print('maximum relative error: ', max_relative_error)
```

Comment:
I don't see `from __future__ print_function` in this file, so wouldn't this print a tuple in Python 2?

Reply:
Good catch. That does print as a tuple in Python 2 which isn't ideal. 

I've added the import to this file and two other affected files.

Reply:
How about adding a fixed `__future__` preamble to every file? That will ensure consistent usage.

After a quick scan over the repo, it looks like there are many more unaddressed cases. This includes both Python 2 style prints (even in a file partially changed in this PR: [here](https://github.com/apple/coremltools/pull/36/files#diff-0879695b4a0279a2d0989173248f2398R1653)). There are also more Python 3 prints without `__future__`, such as in `models/utils.py`.

A 2to3 scan also picked up on occurrences of `iteritems`, `unicode` and other things not available in py3.

Reply:
I've removed the two `iteritems` from the unit tests. Thanks, that's another good catch.

The `unicode` usage and the print statements in `models/utils.py` are in evaluation code paths. Evaluation is intentionally not covered in this PR.

I'm not sure I understand what you mean by a `__future__` preamble.

Reply:
I think adding that to all files is redundant. Long term we probably shouldn't even be using print statement. We should be logging these message instead.